docs/source/development/CDFs/cdf_requirements.rst

```diff
@@ -0,0 +1,343 @@
+
+In the case of the DEPEND_0 timetag variable, DELTA_PLUS_VAR and DELTA_MINUS_VAR together with the timetag identify the time interval over which the data was sampled, integrated, or otherwise regarded as representative of. DELTA_PLUS_VAR and DELTA_MINUS_VAR variables require FIELDNAM, UNITS and SI_CONVERSION attributes; in principle, these could differ from those of the DEPEND_i parent. They also require VAR_TYPE=SUPPORT_DATA. Other standard attributes might be helpful.
+
+Required Support Attributes
+----------------------------
+Variables appearing in a data variable's DEPEND_i attribute require a minimal set of their own attributes to fulfill their role in supporting the data variable.
+
+* CATDESC
+* DEPEND_0 (if time varying)
+* FIELDNAM
+* FILLVAL (if time varying)
+* FORMAT/FORM_PTR
+* SI_CONVERSION
+* UNITS/UNIT_PTR
+* VALIDMIN (if time varying)
+* VALIDMAX (if time varying)
+* VAR_TYPE = “support_data”
+
+These attributes are otherwise the same as described in the above section for data variables
+
+******************************
+Variable Naming Convention
+******************************
+
+Data Variables
+===============
+
+IMAP data variables must adhere to the following naming conventions:
+
+instrument_parameter[_coordinateSystem][_parameterQualifier][_subModeLevel][_Mode][_DataLevel]
+
+An underscore is used to separate different fields in the variable name.  It is strongly recommended that variable named employ further fields, qualifiers, and information designed to identify unambiguously the nature of the variable, instrument mode, and data processing level, with sufficient detail to lead the user to the unique source file which containers the variable.  It is recommended to follow the order shown below
+
+These variable names may only include lower-case letters, numbers, underscores, and hyphens.  No upper-case letter or other special characters are allowed.
+
+Required
+---------
+* instrument - the instrument acronym
+* parameter - a short representation of the physical parameter held in the variable
+
+Optional
+---------
+* coordinateSystem - an acronym for the coordinate system in which the parameter is set
+* parameterQualifier - parameter descriptor, which may include multiple components separated by a "_" as needed (e.g. "pa_0" indicates a pitch angle of 0)
+* subModeLevel - Qualifiers to include mode and data level information supplementary to the next to fields
+* mode - The mode of the instrument
+* dataLevel - The data level in the "Data_type" attribute field
+
+
+Support Data Variables
+=======================
+
+Support data variable names must begin with a letter and can contain numbers and underscores, but no other special characters. Support data variable names need not follow the same naming convention as Data Variables (5.1.1) but may be shortened for convenience.
+
+
+******************************
+File Naming Convention
```

Comment:
I'm wondering if some of this is getting into the `style-guide` territory? I'm not sure where I would expect things to live though.

Reply:
We could add a link to this section the `Naming Conventions` doc in the style guide.